package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": [
+    "dist/",
+    "typings/jquery-autocomplete/",
+    "readme.md"
+  ],
+  "dependencies": {
+    "jquery": "*"
+  },
+  "shim": {
+    "jquery.autocomplete": ["jquery"]
```

Comment:
I have tested this with both the latest stable and beta releases of JSPM and the `"shim"` section is not necessary as this is a well behaved package that only binds to a global `jQuery` as a last resort. Simply having jquery as a dependency cause the right behavior in both 2.2.4 and 3.1.1 releases